src/Handler/EasyHandle.php

```diff
@@ -51,10 +51,24 @@ public function createResponse()
         // HTTP-version SP status-code SP reason-phrase
         $startLine = explode(' ', array_shift($this->headers), 3);
 
+        $headers = \GuzzleHttp\headers_from_lines($this->headers);
+        $normalizedKeys = array_combine(array_map('strtolower', array_keys($headers)), array_keys($headers));
```

Comment:
Here, we can leave the original headers, construct the response with the original code, and fix the headers of the built response, taking advantage of the `withHeader` method which is case insensitive.

What do you think?